src/Cli/Microsoft.DotNet.Cli.Utils/Polyfills.cs

```diff
@@ -3,7 +3,9 @@
 
 #if NET472
 
+#pragma warning disable IDE0130 // Namespace does not match folder structure
 namespace System.Runtime.CompilerServices;
+#pragma warning restore IDE0130 // Namespace does not match folder structure
```

Comment:
This is a special case as this enables a compiler feature. So, it needs to exist in this namespace to work properly.